packages/story-editor/src/app/media/utils/getResourceFromMedia3p.js

```diff
@@ -197,6 +197,10 @@ function getVideoUrls(m) {
 }
 
 function sortMediaBySize(m, mediaUrls) {
+  if (mediaUrls.length < 1) {
```

Comment:
```suggestion
  // https://github.com/GoogleForCreators/web-stories-wp/issues/12083
  if (mediaUrls.length < 1) {
```

Otherwise this looks good 👍